xml/System.Reflection.PortableExecutable/DebugDirectoryEntryType.xml

```diff
@@ -68,6 +68,15 @@ The debug directory entry of type <xref:System.Reflection.PortableExecutable.Deb
       <MemberValue>2</MemberValue>
```

Comment:
I also noticed this class doesn't have a summary. Can we add?

Reply:
Sure.